include/rocksdb/perf_level_by_bitfield.h

```diff
@@ -0,0 +1,104 @@
+#pragma once
+#include <stdint.h>
+
+namespace rocksdb {
+
+struct PerfLevelByBitField {
+  uint8_t perf_level;
+
+  uint8_t enable_user_key_comparison_count_bit : 1;          // 2
```

Comment:
Maybe we should re-arrange these fields according to the order in perf context structure, instead of according to the level?

Reply:
but in lots of test case it will check the perf_level.

Reply:
I think it doesn't matter. The compiler should do optimizations for us.